_posts/2015-01-04-dates.md

```diff
@@ -7,10 +7,10 @@ fa-icon: calendar
 icon-title: true
 ---
 
-Call for papers available: 12 October 2019
+Call for papers available: 6 November 2022
 
-Call for participation closes: Extended to 30 November 2019
+Call for papers closes: 14 December 2022
 
-Devroom schedule available: 14 December 2019
+Devroom schedule available: 15 December 2022
```

Comment:
I guess we're on a deadline, but a few days for us to revise last minute submissions would be nice :)

Reply:
Should we maybe close on the 15th (Wednesday) and publish the schedule on the 17th (Saturday).